programs/vote_api/src/vote_instruction.rs

```diff
@@ -41,11 +40,11 @@ impl VoteInstruction {
             vec![(vote_id, true)],
         )
     }
-    pub fn new_initialize_account(vote_id: Pubkey, staker_id: Pubkey) -> BuilderInstruction {
+    pub fn new_initialize_account(vote_id: Pubkey) -> BuilderInstruction {
```

Comment:
For the curious, this was the bug. Looks like it stemmed from terminology confusion. The vote account is the staking account.